NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -267,41 +267,7 @@ def read_CORSIKA7(input_file, declination=None):
     zenith, azimuth, magnetic_field_vector = get_angles(corsika, declination)
     energy = corsika['inputs'].attrs["ERANGE"][0] * units.GeV  # Assume fixed energy
 
-    # Create RadioShower to store simulation parameters in Event
```

Comment:
I am not sure if @cg-laser goes along with that. The idea was, that read_CORSIKA7() is the "mother" function from which all other functions are derived.